pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -75,17 +78,20 @@ def __init__(self,
                  ds_low_les  = None,
                  extent_low  = None,
                  ffbin = None,
+                 TurbSimbin = None,
+                 Mannbin=None,
                  mod_wake = 1,
                  yaw_init = None,
                  ADmodel = None,
                  EDmodel = None,
-                 nSeeds = 6,
+                 nSeeds = 1,
```

Comment:
The default number of seeds follow the IEC 61400 standard. If you need to run only one seed for the Mann boxes, then you can specify `nSeeds=1` without the need to change the defaults

Reply:
agreed I was just making it easier for myself but you are correct.